notification/cel.go

```diff
@@ -98,6 +102,60 @@ func (t *celVariables) AsMap() map[string]any {
 		output["new_state"] = t.NewState
 	}
 
+	// Placeholders for commonly used fields of the resource
```

Comment:
this seems to be duplicated, can we just have a duty.GetResourceContext(selectable) ?